src/views/Game/AIReview.tsx

```diff
@@ -959,6 +1194,11 @@ export class AIReview extends React.Component<AIReviewProperties, AIReviewState>
                         </span>
                     </div>
                 }
+                { (data.get("user").is_moderator && ((this.ai_review?.type === "full" ) || (this.ai_review?.type === "fast" )) && this.ai_review?.engine === "katago") &&
```

Comment:
Why check both 'full' and 'fast' here? I think those are the only two values `JGOFAIIReview.type` can have

Reply:
So it's essentially just throwing in a "true" in there I suppose. I think initially I wasn't going to show the table for fast reviews, which was also a comment I had received. 

That is just because the data won't be as accurate, but it's the same with the AI graph itself and we do show that also. 

I guess we can take out that part.

Reply:
```suggestion
                { (data.get("user").is_moderator && this.ai_review?.engine === "katago") &&
```